app/uk/gov/hmrc/pillar2/controllers/RegistrationController.scala

```diff
@@ -54,15 +50,21 @@ class RegistrationController @Inject() (
   }
 
   def withoutIdFmRegistrationSubmission(id: String): Action[AnyContent] = authenticate.async { implicit request =>
-    implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
-    logger.info("[Session ID: ${SessionIdHelper.sessionId(hc)}] - Calling Registration Submission without FmId")
     getUserAnswers(id).flatMap { userAnswer =>
       dataSubmissionService
         .sendNoIdFmRegistration(userAnswer)
         .map(handleResult)
     }
   }
 
+  def registerNewFilingMember(id: String): Action[AnyContent] = authenticate.async { implicit request =>
```

Comment:
rename method to registerNoIdNewFilingMember to distinguish between NoId and Id journeys for new filing member.